_about/ROADMAP.md

```diff
@@ -0,0 +1,64 @@
+# Q3 2020 Roadmap
+
+Each quarter the team will highlight areas of focus for our work and upcoming research.
+ 
+We select items to include in the roadmap from community issues and internal priorities. When community pull requests exist for an item, we will prioritize working with the original authors to include their contributions. If the author can no longer take on the implementation, HashiCorp may complete any additional work needed. 
+
+Each release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. 
+
+To make contribution easier, we’ll be using the [`Help Wanted`](https://github.com/hashicorp/terraform-provider-kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) tag to point to issues we’d like to include in this quarter’s series of releases. 
+
+This quarter (July-September ‘20) we will prioritze the following areas of work: 
+
+## Currently In Progress
+
+### 2.0.0 
+
+Major version releases include code removals, deprecations, and breaking changes. A corresponding “upgrade guide” will be published alongside the release. 
+
+In the 2.0.0 release we'll focus on three areas:
+ - Upgrading to [Terraform Plugin SDK v2](https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html)
+ - [Removing support for Terraform 0.11](https://github.com/hashicorp/terraform-provider-kubernetes/issues/905)
+ - [Changing provider credentialling]
+
+### Coverage
```

Comment:
I think we could add some context here to describe how we're planning to approach improvements according to these themes. Could just be a line like "In an attempt to focus on specific areas of the user experience when improving the provider, we plan to identify and focus on a set of themes each quarter. Here are the 3 themes we plan to work on this quarter." or something like that?

Reply:
Good idea.  Let me know if you like the wording I added.